packages/webpack/src/config/base.js

```diff
@@ -215,7 +216,7 @@ export default class WebpackBaseConfig {
   alias () {
     return {
       ...this.buildContext.options.alias,
-      consola: require.resolve(`consola/dist/consola${this.isServer ? '' : '.browser'}.js`)
+      consola: require.resolve(`consola/${this.isServer ? 'dist/consola' : 'src/browser'}.js`)
```

Comment:
Because consola server build bundled dependencies like `dayjs` which are not in deps of Nuxt, so use `dist/consola.js` for server build.